backend/test/auth_test.py

```diff
@@ -0,0 +1,195 @@
+from datetime import datetime, timedelta, timezone
+import pytest
+import jwt
+from backend.models.team import Team
+from backend.services.exceptions import (
+    InvalidCredentialsException,
+    ResourceNotAllowedException,
+    ResourceNotFoundException,
+)
+from backend.models.auth import TokenData
+from sqlmodel import Session
+from .fixtures import auth_svc, team_svc
+from .fake_data.auth import create_good_teams
+from backend.config import SECRET_KEY, ALGORITHM, ACCESS_TOKEN_EXPIRE_MINUTES
+
+
+def test_authenticate_team_success(auth_svc, create_good_teams):
+    """Tests a successful authentication"""
+    session = create_good_teams
+    team = session.get(Team, 1)
+    token = auth_svc.authenticate_team(team.name, team.password)
+    assert token.token_type == "bearer"
+    assert token.access_token is not None
+
+    decoded_token = jwt.decode(
+        token.access_token, key=SECRET_KEY, algorithms=[ALGORITHM]
```

Comment:
Just set `algorithm` here and everywhere in this file to `HS256`.

Reply:
Me and Nick just did that. You can attempt to pull again because I've already made the correct changes!